karspexet/economy/views.py

```diff
@@ -11,3 +12,16 @@ def overview(request):
         'shows': shows,
         'user': request.user,
     })
+
+@staff_member_required
+def show_detail(request, show_id):
+    show = Show.objects.get(pk=show_id)
+    taken_seats = [ticket.seat_id for ticket in Ticket.objects.filter(show=show).all()]
```

Comment:
You can use `values_list` here to avoid the list comprension. `Ticket.objects.filter(show=show).values_list('seat_id', flat=True)`

Reply:
Ah, nice!